src/agenteval/cli.py

```diff
@@ -130,6 +130,12 @@ def verify_git_reproducibility() -> None:
         )
 
 
+def check_using_local_litellm_model_cost_map():
+    assert (
```

Comment:
Suggest `if ...: raise click.ClickException(...)`.  Assert statements [may be disabled in production](https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement), e.g. when running python with `-O` (not common but I assume users will cover the breadth of possible python setups)